### PR TITLE
GPII-1886: Update dependencies and improve tests...

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [GPII-1886](https://issues.gpii.net/browse/GPII-1886) for details.

### CODE REVIEW

**Comment by the-t-in-rtf** [comment]
All follow-up issues have been addressed and this is ready for (a hopefully brief) review.

File: package.json [hunk 1]
```diff
@@ -5,19 +5,20 @@
...
+    "pretest": "node_modules/rimraf/bin.js coverage instrumented",
     "test": "node node_modules/testem/testem.js ci --skip PhantomJS,Chromium,Safari"
   },
   "devDependencies": {
-    "eslint": "2.11.1",
+    "eslint": "3.2.2",
     "grunt": "1.0.1",
-    "grunt-eslint": "18.1.0",
-    "grunt-jsonlint": "1.0.4",
+    "fluid-grunt-eslint": "18.1.1",
```
Comment (amb26) [inline package.json:14]: Linting doesn't function because the gruntfile wasn't updated for this dependency change

**Comment by the-t-in-rtf** [comment]
The linting issues mentioned earlier have been addressed.  This is one of the further pulls up "the stack", it would be helpful to finish this up and release/publish a version.

**Comment by the-t-in-rtf** [comment]
This is a lower priority than gpii-express, but is still very high up "the stack".  However, it has the virtue of not depending on anything, including gpii-express, so it's safe to merge and publish independently of gpii-express.

File: package.json [hunk 2]
```diff
@@ -5,19 +5,20 @@
   "license": "BSD-3-Clause",
   "repository": "https://github.com/GPII/gpii-binder",
   "scripts": {
+    "pretest": "node_modules/rimraf/bin.js coverage instrumented",
```
Comment (amb26) [inline package.json]: This "executable directive" doesn't function on Windows

Comment (the-t-in-rtf) [reply package.json]: I updated it to use `node node_modules/rimraf/bin.js`, which should work anywhere.

**Pull request merged**